src/context.cpp

```diff
@@ -470,6 +507,33 @@ namespace librealsense
     {
         std::shared_ptr< realdds::dds_device > _dds_dev;
 
+        rs2_video_stream to_rs2_format( const realdds::dds_video_stream::profile & profile )
```

Comment:
The function retunes a video stream profile but it's called `to_rs2_format`,
IMO it's confusing especially when we have a `format` attribute inside the profile

Reply:
Renamed `to_rs2_video_stream`